types/helpers/helpers.color.d.ts

```diff
@@ -0,0 +1,33 @@
+export function color(value: CanvasGradient): CanvasGradient;
+export function color(value: CanvasPattern): CanvasPattern;
+
+export interface ColorModel {
```

Comment:
@kurkle does your color lib have types? If so, we can use those

Reply:
It has. Those can probably be improved though.